thoth/storages/graph/postgres.py

```diff
@@ -168,7 +168,23 @@ def parse_python_solver_name(solver_name: str) -> dict:
 
     def get_analysis_metadata(self, analysis_document_id: str) -> dict:
         """Get metadata stored for the given analysis document."""
-        raise NotImplementedError
+        query = (
+            self._session.query(PackageExtractRun)
+            .filter(PackageExtractRun.analysis_document_id == analysis_document_id)
+            .with_entities(
+                PackageExtractRun.datetime,
+                PackageExtractRun.analysis_document_id,
+                PackageExtractRun.package_extract_name,
+                PackageExtractRun.package_extract_version
+            )
+        )
+        query_result = query.one()
```

Comment:
Don't we add an error if the query is empty? https://github.com/thoth-station/storages/blob/39206123de88916cbd4143b45b16796cad2bd4bc/thoth/storages/graph/dgraph.py#L303

Reply:
`one()` raises an error if no results are found. We should map it to our error though.

Reply:
do you want to change it?

Reply:
Adjusted in aaf5f0f :+1: